airbyte-integrations/connectors/source-rarible/manifest.yaml

```diff
@@ -0,0 +1,1739 @@
+version: 5.10.2
+
+type: DeclarativeSource
+
+description: >-
+  This connector integrates Rarible's API with Airbyte, enabling seamless data
+  synchronization of NFT activities, collections, and ownership across multiple
+  blockchains. It supports fetching  ownership, orders and other essential
+  details to facilitate real-time updates and analytics for NFT marketplaces.
+
+check:
+  type: CheckStream
+  stream_names:
+    - nfts
+
+definitions:
+  streams:
+    nfts:
+      type: DeclarativeStream
+      name: nfts
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /v0.1/items/all
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - items
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/nfts"
+    collections:
+      type: DeclarativeStream
+      name: collections
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /v0.1/collections/all
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - collections
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/collections"
+    statistics:
+      type: DeclarativeStream
+      name: statistics
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: v0.1/data/collections/{{ config['collection_id'] }}/stats
```

Comment:
Why not use collection stream as parent of this one?

Reply:
yes I'll update it . At that time I didnt knew about the parent stream thing

Reply:
👍